WalletWasabi.Fluent/Views/Wallets/Home/TilesView.axaml

```diff
@@ -0,0 +1,35 @@
+<UserControl xmlns="https://github.com/avaloniaui"
```

Comment:
Why new name, it's not generic control but wallet tiles, I would leave old name WalletTiles I think

Reply:
I mean `TilesView` gives the feeling there is a `TilesViewModel` somewhere, while there is not. I agree with @wieslawsoltes on the old name suits better.